Extension/src/nativeStrings.json

```diff
@@ -3,6 +3,7 @@
     "include_not_found_in_browse_path": "Include file not found in browse.path.",
     "update_browse_path": "Edit \"browse.path\" setting",
     "add_to_include_path": "Add to \"includePath\": {0}",
+    "add_missing_include_path": "Add {0}",
```

Comment:
It seems like this should be
```json
    "add_missing_include_path": {
        "text": "Add '{0}'",
        "hint": "{0} is C++ code to add, such as '#include <string>'"
    },
```
since there could be a potential translation ambiguity with just "Add {0}".

Reply:
...and I'm not sure yet if it should be `{0}` or `'{0}'` or `\"{0}\"`

Reply:
VS uses `'{0}'` so I'd say go with that.